contributing/documentation/standards.rst

```diff
@@ -137,8 +144,13 @@ Files and Directories
 English Language Standards
 --------------------------
 
-* **English Dialect**: use the United States English dialect, commonly called
-  `American English`_.
+Symfony documentation uses the United States English dialect, commonly called
+`American English`_. Disputes over vocabulary will always be resolved using the
+`American English Oxford Dictionary`_ reference. Disputes over neologisms not
+included yet in the dictionary will be resolved by Symfony Documentation maintainers.
```

Comment:
OK then. I've removed this section and left just the reference to the dictionary.